edpm_ansible/ansible_plugins/modules/container_config_hash.py

```diff
@@ -209,29 +252,28 @@ def _update_hashes(self):
                 continue
             startup_config_json = json.loads(self._slurp(config))
             config_volumes = self._match_config_volumes(startup_config_json)
-            config_hashes = [
-                self._get_config_hash(vol_path) for vol_path in config_volumes
-            ]
-            config_hashes = filter(None, config_hashes)
-            if 'environment' in startup_config_json:
-                old_config_hash = startup_config_json['environment'].get(
-                    'EDPM_CONFIG_HASH', '')
-            if config_hashes is not None and config_hashes:
-                config_hash = '-'.join(config_hashes)
-                if config_hash == old_config_hash:
-                    # config doesn't need an update
-                    continue
-                self.module.debug(
-                    'Config change detected for {}, new hash: {}'.format(
-                        cname,
-                        config_hash
-                    )
-                )
-                if 'environment' not in startup_config_json:
-                    startup_config_json['environment'] = {}
-                startup_config_json['environment']['EDPM_CONFIG_HASH'] = (
-                    config_hash)
-                self._update_container_config(config, startup_config_json)
+            if config_volumes:
+                config_hashes = [
+                    self._get_config_hash(vol_path) for vol_path in config_volumes
+                ]
+                if 'environment' in startup_config_json:
+                    old_config_hash = startup_config_json['environment'].get(
+                        'EDPM_CONFIG_HASH', '')
+                if config_hashes is not None and config_hashes:
```

Comment:
config_hashes can't be none and is always a list.

Reply:
removed completely config_hashes since it's useless with the new approach.